src/Math/Helper.dfy

```diff
@@ -8,6 +8,27 @@ module Helper {
    Definitions
   ************/
 
+  function ArrayToSeq<T>(a: array<T>): (xs: seq<T>)
```

Comment:
I'd rather not use this function since the conversion is defined in the ref manual. Is this for readability?

Reply:
Yes, I made it explicit for readability. I'll revert.